### DISCUSSION TITLE
Provide pi_bullseye motion package builds

### DISCUSSION THREAD

**Discussion by MichaIng** [Motion-Project/motion#1654] [General]
I know there are open issues regarding legacy RPi camera support being disabled by default on RPi OS since Bullseye. But this can be generally worked around and also other cameras can be used on RPi. As the build system generally supports Raspbian Bullseye builds, would you mind to add a `pi_bullseye` package build to the current release 4.5.1?

**Comment by Mr-Dave** [comment]
I've put out a Bulleye build for the PI

**Reply by MichaIng** [reply]
That is great, many thanks 👍.
